release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -1764,3 +1764,19 @@ link:https://access.redhat.com/errata/RHBA-2018:3745[RHBA-2018:3745] advisory.
 To upgrade an existing {product-title} 3.10 or 3.11 cluster to this latest
 release, see xref:../upgrading/index.adoc#install-config-upgrading-index[Upgrade
 methods and strategies] for instructions.
+
+[[ocp-3-11-RHBA-2018-3688]]
+=== RHBA-2018:3688 - {product-title} 3.11 Package Updates for IBM POWER
+
+Issued: 2018-12-13
+
+{product-title} release 3.11 is now available updates to packages for ppc64le.
```

Comment:
@ahardin-rh is there a word missing in this line (1773)?

Reply:
@jboxman Whoops! Good catch. I opened https://github.com/openshift/openshift-docs/pull/13157 to fix that.